Task3/MoneyManager/SettingsClassLibrary/Startup.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Data;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace SettingsClassLibrary
```

Comment:
As discussed, let's rename this project to `MoneyManager.CompositionRoot` and add here all DI settings (through extension method).